package.json

```diff
@@ -9,6 +9,7 @@
     "build": "parcel build"
   },
   "devDependencies": {
+    "@parcel/transformer-sass": "^2.2.1",
```

Comment:
Qual foi o motivo de instalar esse pacote? 🤔

Reply:
No início eu achei que tinha que usar o sass, aí eu tava tentando compilar o scss pra css

Reply:
já retirei das dependências